src/lib/pay-request/services/admin_users/client.ts

```diff
@@ -97,7 +97,7 @@ export default class AdminUsers extends Client {
 
     findByEmail(email: string) : Promise<User | undefined> {
       return client._axios
-        .post('/v1/api/users/find', { username: email })
+        .post('/v1/api/users/find', { username: email, email: email })
```

Comment:
Can we get rid of the username field?
Just use email?

Reply:
not in the first round, next iteration. adminusers is still expecting username. once that is changed to email, we can remove it from toolbox